ecs-cli/modules/cli/regcreds/create_task_execution_role.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2015-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package regcreds
+
+import (
+	"time"
+
+	iamClient "github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/iam"
+	kmsClient "github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/kms"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/iam"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	assumeRolePolicyDocString  = `{"Version":"2008-10-17","Statement":[{"Sid":"","Effect":"Allow","Principal":{"Service":"ecs-tasks.amazonaws.com"},"Action":"sts:AssumeRole"}]}`
+	policyDescriptionFmtString = "Policy generated by the ecs-cli for role %s"
```

Comment:
I also felt like it seemed unusual to have this be a constant, and it might make the code easier to read if this string was placed where its actually used